testproject/django_file_form_example/static/example_form_custom_widget.js

```diff
@@ -0,0 +1,43 @@
+const eventEmitter = new EventEmitter3();
+
+eventEmitter.on('addUpload', ({ element, fieldName, metaDataField, upload }) => {
+  function descriptionChanged(evt) {
```

Comment:
Put the `descriptionChanged` here so `fieldName` and other values can be accessed.

Reply:
```
eventEmitter.on('addUpload', ({ element, fieldName, metaDataField, upload }) => {
  function descriptionChanged(evt) {
```

My limited knowledge on JavaScript shouts that `descriptionChanged` happens long after `addUpload` is triggered and  this cannot be safe.... but I trust you here. :-)